src/components/AppProvider/utilities/withAppProvider/withAppProvider.tsx

```diff
@@ -46,7 +46,7 @@ function withScrollable<P, T>(WrappedComponent: ReactComponent<P, T>) {
                 stickyManager: this.stickyManager,
               }}
             >
-              <WrappedComponent {...this.props} />
+              <WrappedComponent {...this.props as any} />
```

Comment:
On `line 115` return the component as `React.ComponentClass<OwnProps> & C` so this won't have any effect on the return type